docs/develop/dotnet/set-up.mdx

```diff
@@ -0,0 +1,550 @@
+---
+id: set-up-your-local-dotnet
+title: Set up your local with the .NET SDK
+sidebar_label: Quickstart - Setup
+description: Configure your local development environment to get started developing with Temporal
+keywords:
+  - set up
+  - getting started
+  - dotnet sdk
+tags:
+  - developer guide
+  - dotnet
+  - setup
+  - getting started
+hide_table_of_contents: true
+---
+
+import { SetupSteps, SetupStep, CodeSnippet } from "@site/src/components/elements/SetupSteps";
+import { InfoBox } from "@site/src/components/elements/InfoBox";
+import { CallToAction } from "@site/src/components/elements/CallToAction";
+
+# Quickstart - Setup
+
+Configure your local development environment to get started developing with Temporal.
+
+<SetupSteps>
+<SetupStep code={
+  <>
+    <p>The .NET SDK requires .NET 6.0 or later.</p>
+    <p>Install .NET by following the <a href="https://dotnet.microsoft.com/download/dotnet/6.0">official .NET instructions</a>.</p>
```

Comment:
Should we go ahead and link to `https://dotnet.microsoft.com/en-us/download/dotnet/9.0` since `6.0` has reached EOL?